.github/workflows/pr-title.yml

```diff
@@ -7,21 +7,18 @@ on:
 jobs:
   lint-pr-title:
     runs-on: ubuntu-latest
-
     steps:
-      - name: 🔧 Setup Bun
-        uses: oven-sh/setup-bun@4bc047ad259df6fc24a6c9b0f9a0cb08cf17fbe5 # v2
-        with:
-          bun-version-file: ".bun-version"
-
       - name: 📥 Checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           # Only fetch the config file from the repository
           sparse-checkout: commitlint.config.js
 
       - name: 📦 Install dependencies
-        run: bun i @commitlint/config-conventional
+        run: npm install --global @commitlint/config-conventional commitlint
```

Comment:
Why not ["bun install --global"](https://bun.sh/docs/cli/install#global-packages)? I had the feeling that bun was faster than npm.

Reply:
> Why not ["bun install --global"](https://bun.sh/docs/cli/install#global-packages)? I had the feeling that bun was faster than npm.

It is not installed by default, and if it is installed, it will take ~3s more time

Reply:
ok yeah let's skip it for this workflow. Node and npm are fine for this as well.